<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MÃºsica Visual</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="4" column="3" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Resolution:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="5">
     <widget class="QSpinBox" name="spb_res_y">
      <property name="maximum">
       <number>4096</number>
      </property>
      <property name="value">
       <number>480</number>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QDoubleSpinBox" name="dsb_fps">
      <property name="maximum">
       <double>60.000000000000000</double>
      </property>
      <property name="value">
       <double>30.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="spb_res_x">
      <property name="maximum">
       <number>4096</number>
      </property>
      <property name="value">
       <number>720</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="3">
     <widget class="QPushButton" name="pb_animation">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Animation!</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QToolButton" name="toolButton">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3" colspan="3">
     <widget class="QPushButton" name="pb_process">
      <property name="text">
       <string>Process</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3" colspan="3">
     <widget class="QPushButton" name="pb_noteBlocks">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Note blocks</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3" colspan="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Frames per second:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="edt_videoOutput"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Video output:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="8" colspan="3">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>516</width>
         <height>210</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPlainTextEdit" name="plainTextEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Monospace</family>
           </font>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionImport_MIDI_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionTracks"/>
    <addaction name="actionRendering_Properties"/>
    <addaction name="separator"/>
    <addaction name="actionSave_settings_as"/>
    <addaction name="actionLoad_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHow_does_it_work"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSqueeze_tracks"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionImport_MIDI_File">
   <property name="text">
    <string>Import MIDI File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionTracks">
   <property name="text">
    <string>Tracks...</string>
   </property>
  </action>
  <action name="actionRendering_Properties">
   <property name="text">
    <string>Rendering Properties...</string>
   </property>
  </action>
  <action name="actionSave_settings_as">
   <property name="text">
    <string>Save settings as...</string>
   </property>
  </action>
  <action name="actionLoad_settings">
   <property name="text">
    <string>Load settings...</string>
   </property>
  </action>
  <action name="actionHow_does_it_work">
   <property name="text">
    <string>How does it work?</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionSqueeze_tracks">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Squeeze tracks</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
